packages/kbn-optimizer/limits.yml

```diff
@@ -8,7 +8,7 @@ pageLoadAssetSize:
   charts: 95000
   cloud: 21076
   console: 46091
-  core: 432925
+  core: 450579
```

Comment:
Using the `path` module is what caused the bundle to grow in size like this, I think if we want to get rid of the `url.resolve()` call we should instead just write a simple function that takes care of ensuring there is a single `/` between the two components and avoid raising the bundle limit any further than it already is.

Reply:
Hi @spalger thanks for taking the time to review, I agree, I've simplified it now to just use string concatenation, this does rely on the fact that the base url ends with "/" but I can't imagine this would trip anyone up? Let me know what you think 👍